MODULE.bazel

```diff
@@ -0,0 +1,17 @@
+module(name = "com_github_y_scope_clp_ffi_go", version = "0.0.5-beta")
```

Comment:
> Do we need a WORKSPACE? (Would it help projects which haven't migrated to Bzlmod?)

Afaik, this would only enable building ffi-go itself through old versions of bazel, which isn't necessary to support (on its own ffi-go doesn't really do anything).

> REPO.bazel does seem good to have for specifying the OSS license for all sources: https://bazel.build/external/overview#repo.bazel Do we want to add such?

As discussed, exploring https://bazelbuild.github.io/rules_license/latest.html